project/Dependencies.scala

```diff
@@ -40,4 +40,12 @@ object Dependencies {
       "com.google.jimfs"  %  "jimfs"               % "1.1"  % Test  // ApacheV2
     )
   )
+
+  val Sqs = Seq(
+    libraryDependencies ++= Seq(
+      "com.amazonaws"   % "aws-java-sdk-sqs"    % "1.11.51",          // ApacheV2
+      "org.elasticmq"  %% "elasticmq-rest-sqs"  % "0.10.1"    % Test  // ApacheV2
```

Comment:
Oops, my bad. I didn't know that the authors have invented a new individual style. Keep it, I might try to change all of them later, if I have too much time ;-)